.browserslistrc

```diff
@@ -3,11 +3,10 @@
 >= 1%
 last 1 major version
 not dead
-Chrome >= 45
-Firefox >= 38
-Edge >= 12
-Explorer >= 10
-iOS >= 9
-Safari >= 9
-Android >= 4.4
-Opera >= 30
+Chrome >= 70
```

Comment:
I wouldn't go there. IIRC the Google search bot is based on Chrome 47/48 or something like that. I need to find the source for this though.

Reply:
Found it: https://developers.google.com/search/docs/guides/rendering

>Googlebot uses a web rendering service (WRS) that is based on Chrome 41 (M41)

Maybe by the time v5 stable is out this has changed, but until then, we shouldn't do this.

Reply:
FYI *"The new evergreen Googlebot [...] now runs the latest Chromium rendering engine (74 at the time of this post) [...] Moving forward, Googlebot will regularly update its rendering engine"*

Source: https://webmasters.googleblog.com/2019/05/the-new-evergreen-googlebot.html

And https://developers.google.com/search/docs/guides/fix-search-javascript does not specify anymore the Chrome version used by Googlebot